tests/src/test_generate_mapping.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+import config
+from google.cloud import bigquery
+import generate_td_to_bq_mapping as gm
+import os
+from  pathlib import Path
+import utils
+from google.api_core.exceptions import NotFound, Forbidden, BadRequest, ServiceUnavailable, Conflict, TooManyRequests
+
+class TestObjectMapping:
+
+    def test_get_created_tables_and_views_success(self,
+                                                  create_sql_file):
+        file_path = "test_file.sql"
+        sql_path = Path(file_path)
+        get_tables_and_views = gm.get_created_tables_and_views(sql_path)
+        assert type(get_tables_and_views) == type(list())
+
+    def test_get_created_tables_and_views_fail_due_to_invalid_path(self):
+        with pytest.raises(AttributeError):
+            gm.get_created_tables_and_views(sql_path="file.fake")
+
+    def test_write_table_mapping_to_bq_success(self,
+                                               bq_client,
+                                               create_and_delete_table):
+        client = bq_client
+        table_map = {"key1": "item1", "key2": "item2", "key3": "item3"}
+        table_mappings = write_table_mapping_to_bigquery_test(client=client,
+                                                              table_map=table_map)
+        print(table_mappings, type(table_mappings))
+        assert table_mappings == None
```

Comment:
Write a query against the table where this data is stored to ensure the records that we expect have been inserted into the table.